apps/backend/prisma/migrations/20260627120000_event_drafts/migration.sql

```
@@ -0,0 +1,36 @@
+  CONSTRAINT "event_drafts_pkey" PRIMARY KEY ("id")
+);
+
+ALTER TABLE "event_drafts"
+  ADD CONSTRAINT "event_drafts_sourceEventId_fkey"
+  FOREIGN KEY ("sourceEventId")
+  REFERENCES "events"("id")
+  ON DELETE CASCADE
+  ON UPDATE CASCADE;
+
+CREATE INDEX "event_drafts_sourceEventId_idx"
+ON "event_drafts"("sourceEventId");
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)